pages/index.py

```diff
@@ -265,3 +247,4 @@ def __init__(self):
             ServiceSection(),
             NewsSection(),
             ContactSection()]
+
```

Comment:
Oh wait, I just read the second part of your comment again. I'll drop the newline.